distribution/docs/src/main/resources/content/_devComponents/editing-docs.adoc

```diff
@@ -0,0 +1,99 @@
+:title:  Contributing to Documentation
+:type: developingComponent
+:status: published
+:link: _contributing_to_documentation
+:summary: Updating documentation.
+:order: 99
+
+${branding} documentation is included in the source code, so it is edited and maintained in much the same way.
+
+`src/main/resources`
+
+.Documentation Directory Structure and Contents
+[cols="1m,3" options="headers"]
+|===
+|Directory
+|Contents
+
+|content
+|Asciidoctor-formatted files containing documentation contents and the header information needed to organize them.
+
+|images
+|Screenshots, icons, and other image files used in documentation.
+
+|templates
+|Template files used to compile the documentation for display.
+
+|jbake.properties
+|Properties file defining content types and other parameters.
+|===
+
+==== Editing Existing Documentation
+
+Update existing content when code behavior changes, new capabilities are added to features, or the configuration process changes.
+Content is organized within the `content` directory in sub directories according the audience and purpose for each section of the documentation.
```

Comment:
Typo: `sub directories according the audience` should have `to` between `according` and `the`.